data/clients.json

```diff
@@ -124,6 +124,10 @@
 			"name": "Certhub",
 			"url": "https://certhub.io/"
 		}
```

Comment:
I believe you're missing a `,` here. The JSON in this branch is invalid and [failing in CI](https://travis-ci.org/letsencrypt/website/builds/556739712).